public/components/context_menu/context_menu.js

```diff
@@ -87,7 +91,7 @@ const generateInContextReport = async (
   };
 
   fetch(
-    `${getApiPath()}/reporting/generateReport?${new URLSearchParams(
+    `../${getApiPath()}/reporting/generateReport?${new URLSearchParams(
```

Comment:
instead of this, try adding `if (window.location.href.includes('/data-explorer/discover/')) return '../../../api'` in line 339 (above the other if)